components/DesktopMenu/DesktopMenu.tsx

```diff
@@ -25,14 +26,32 @@ const DesktopMenu = () => {
   return (
     <div className="flex flex-row text-sm uppercase font-quicksand gap-x-12">
       <div className="flex items-center pr-4">
-        <a href="https://reserve.cre8ors.com/" target="_blank" rel="noreferrer">
+        <Link href="https://reserve.cre8ors.com/" target="_blank" rel="noreferrer">
           <div className="font-bold cursor-pointer dark:text-white text-black">Reserve List</div>
-        </a>
+        </Link>
       </div>
       <div className="flex items-center font-quicksand font-bold">
-        <div className="pr-2 text-[#9C9C9C]">light</div>
+        <button
+          type="button"
+          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+          onClick={() => {
+            setIsDarkMode(false)
+            onChangeThemeConfig("light")
+          }}
+        >
+          light
+        </button>
         <ToggleButton onClick={onToggle} value={isDarkMode} id="light_dark_switch" />
-        <div className="pl-2 text-[#9C9C9C]">dark</div>
+        <button
+          type="button"
+          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+          onClick={() => {
+            setIsDarkMode(true)
+            onChangeThemeConfig("dark")
+          }}
+        >
```

Comment:
this code is repeated between both buttons right?
- please avoid defining functions in react code.
- please create a shared function used by both the `light` and `dark` button so you do not repeat this code.

let's focus on writing reusable components.

principle: [DRY - do not repeat yourself](https://docs.getdbt.com/terms/dry#:~:text=DRY%20code%20means%20you%20get,and%20energy%20on%20tedious%20syntax.)